src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -114,6 +143,136 @@ private TimeZoneInfo(byte[] data, string id, bool dstDisabled)
             ValidateTimeZoneInfo(_id, _baseUtcOffset, _adjustmentRules, out _supportsDaylightSavingTime);
         }
 
+        // Helper function that builds the value backing the DisplayName field from gloablization data.
+        private static void GetFullValueForDisplayNameField(string timeZoneId, TimeSpan baseUtcOffset, string? standardName, CultureInfo uiCulture, ref string? displayName)
```

Comment:
>standardName [](start = 111, length = 12)

does this paramter ever used in this method?

Reply:
Good catch.  Was an artifact from an earlier iteration.  Will remove.